motion/p5js/air-dancer/index.md

```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Marius Claßen
+author-url: https://github.com/mariusclassen
+title: Air Dancer
+tags: [P5.js, motion, law of connected elements]
```

Comment:
Should be

Law of Connected Element

Sorry this is a inconsistency I introduced.